core/src/tangram.h

```diff
@@ -148,6 +150,49 @@ class Map {
 
     void clearDataSource(DataSource& _source, bool _data, bool _tiles);
 
+    // Add a marker object to the map and return a handle to it; _styling is a string of YAML
+    // that specifies a 'draw rule' according to the scene file syntax; The marker will not be
+    // drawn until it a point, line, or polygon is set for it with one of the functions below.
+    Marker* markerAdd(const char* _styling);
+
+    // Remove a marker object from the map; returns true if the marker handle was found and
+    // successfully removed, otherwise returns false.
+    bool markerRemove(Marker* _marker);
+
+    // Set the geometry of a marker to a point at the given coordinates; markers can have their
+    // geometry set multiple times with possibly different geometry types; returns true if the
+    // marker handle was found and successfully updated, otherwise returns false.
+    bool markerSetPoint(Marker* _marker, LngLat _lngLat);
+
+    // Set the position of a marker with point geometry to ease to a new location over a duration
+    // in seconds with the given EaseType; returns true if the marker handle was found and has a
+    // point geometry, otherwise returns false.
+    bool markerSetPointEased(Marker* _marker, LngLat _lngLat, float _duration, EaseType _ease);
```

Comment:
This function name `markerSetPointEased` is a bit confusing. According to the name it should set the Point and if already set by a previous call, update it using the specified ease.
Already discussed with @blair1618 IRL. We should probably name this to `markerSetEased` and implement easing for all types of marker geometries.